samples/ChatSample/Startup.cs

```diff
@@ -39,6 +39,12 @@ public Startup(IHostingEnvironment env)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+             // if you want use Cors  must add this first
+            //Add Cors
+            services.AddCors(options => options.AddPolicy("AllowAll", p => p.AllowAnyOrigin()
+                                                                 .AllowAnyMethod()
+                                                                  .AllowAnyHeader()
+                                                                  .AllowCredentials()));
```

Comment:
I don't think we would want to have something like this in a sample - it allows requests sent in any form sent from anywhere. I think the best practice is to only allow what is required instead of allow anything.